### PR TITLE
[bugs 1350006, 1350009] Health subpages - Decentralization and Web Literacy

### PR DESCRIPTION
**Pull request by craigcook**
## Description
Adds two new Internet Health subpages; Decentralization and Web Literacy.
Also includes a string update on the Digital Inclusion page with a simpler string that should be easier to localize. Uses the tag `inclusion-donate-tech`
Adds a redirect from /internethealth to /internet-health.

## Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1350006
https://bugzilla.mozilla.org/show_bug.cgi?id=1350009
https://bugzilla.mozilla.org/show_bug.cgi?id=1335569
https://bugzilla.mozilla.org/show_bug.cgi?id=1361194

## Testing
https://www-demo1.allizom.org/en-US/internet-health/decentralization/
https://www-demo1.allizom.org/en-US/internet-health/web-literacy/

## Checklist
- [x] Requires l10n changes.
- [ ] Related functional & integration tests passing.

### CODE REVIEW

**Comment by craigcook** [comment]
I've pushed this to demo1 but for some reason the two new pages are a 404, though they work locally and there don't seem to be any errors in circle or jenkins, so I dunno what's up with that. Maybe something in urls.py?

**Comment by craigcook** [comment]
Demo fixed. I had a wrong filename on an image. D'oh!

File: bedrock/mozorg/redirects.py [hunk 1]
```diff
@@ -668,4 +668,7 @@ def decider(request, **kwargs):
         'utm_medium': 'collateral',
         'utm_campaign': 'css-grid',
     }),
+
+    # Bug 1361194
+    redirect(r'^internethealth/?$', 'mozorg.internet-health'),
```
Comment (alexgibson) [inline bedrock/mozorg/redirects.py]: Would be good to add a test for this?

File: bedrock/mozorg/templates/mozorg/internet-health/digital-inclusion.html [hunk 2]
```diff
@@ -30,8 +30,10 @@
 <nav id="health-subnav" class="health-subnav">
   <ul class="content">
     <li><a href="{{ url('mozorg.internet-health.privacy-security') }}">{{ _('Privacy &amp; Security') }}</a></li>
-    <li class="current"><a href="{{ url('mozorg.internet-health.digital-inclusion') }}">{{ _('Digital Inclusion') }}</a></li>
     <li><a href="{{ url('mozorg.internet-health.open-innovation') }}">{{ _('Open Innovation') }}</a></li>
+    <li><a href="{{ url('mozorg.internet-health.decentralization') }}">{{ _('Decentralization') }}</a></li>
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/internet-health/digital-inclusion.html]: Do these new strings need to be behind l10n tags?

File: bedrock/mozorg/templates/mozorg/internet-health/open-innovation.html [hunk 3]
```diff
@@ -30,8 +30,10 @@
 <nav id="health-subnav" class="health-subnav">
   <ul class="content">
     <li><a href="{{ url('mozorg.internet-health.privacy-security') }}">{{ _('Privacy &amp; Security') }}</a></li>
-    <li><a href="{{ url('mozorg.internet-health.digital-inclusion') }}">{{ _('Digital Inclusion') }}</a></li>
     <li class="current"><a href="{{ url('mozorg.internet-health.open-innovation') }}">{{ _('Open Innovation') }}</a></li>
+    <li><a href="{{ url('mozorg.internet-health.decentralization') }}">{{ _('Decentralization') }}</a></li>
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/internet-health/open-innovation.html]: Same here, new strings need tags?

File: bedrock/mozorg/templates/mozorg/internet-health/privacy-security.html [hunk 4]
```diff
@@ -35,8 +35,10 @@
 <nav id="health-subnav" class="health-subnav">
   <ul class="content">
     <li class="current"><a href="{{ url('mozorg.internet-health.privacy-security') }}">{{ _('Privacy &amp; Security') }}</a></li>
-    <li><a href="{{ url('mozorg.internet-health.digital-inclusion') }}">{{ _('Digital Inclusion') }}</a></li>
     <li><a href="{{ url('mozorg.internet-health.open-innovation') }}">{{ _('Open Innovation') }}</a></li>
+    <li><a href="{{ url('mozorg.internet-health.decentralization') }}">{{ _('Decentralization') }}</a></li>
```
Comment (alexgibson) [inline bedrock/mozorg/templates/mozorg/internet-health/privacy-security.html]: And here?

**Comment by alexgibson** [comment]
This small link contrast issue is present on some existing sub pages, but it really shows on the web literacy page. Think we could find something nicer that fits all sub pages?

screen shot 2017-05-04 at 10 39 03 am

File: bedrock/mozorg/redirects.py (hunk 1 above)
Comment (craigcook) [inline bedrock/mozorg/redirects.py]: Done.

File: bedrock/mozorg/templates/mozorg/internet-health/digital-inclusion.html (hunk 2 above)
Comment (craigcook) [inline bedrock/mozorg/templates/mozorg/internet-health/digital-inclusion.html]: Good catch. Added the tag `health-decentralization-and-literacy` (just for the subnav on existing pages).

**Pull request merged**